### ISSUE TITLE
Tests for displayPublicKey misuse str.strip

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5923]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#5923|
|Type|defect|
|Created|2012-08-28 11:04:29Z|

In r35420 tests were introduced using strip with an argument, instead of just removing that last part of the string (I think).

Attachments:

* [5923-test_ckeygen.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/e59/e59b98cc4d381db2a12c26ecb0bc1c2c8148461b/337fb8353394465ee3ad57ab446bf2e2275d60c4.diff) (1848 bytes) - added by ltaylor.volks on 2012-08-28 18:28:36Z - Fix incorrect usage of str.strip()

<details><summary>Searchable metadata</summary>

```
trac-id__5923 5923
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__easy easy
time__1346151869000000 1346151869000000
changetime__1347209510000000 1347209510000000
version__None None
owner__ 
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="ltaylor.volks's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|ltaylor.volks commented|
|-|-|

This should work regardless of what comment is added to the public key

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [35562]) Remove misuse of str.strip in ckeygen tests.

Author: ltaylor.volks
Reviewer: therve
Fixes: [#5923](https://github.com/twisted/twisted/issues/5923)
